app.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const app = express();
+const { getTopics } = require("./controllers/topics-controllers");
+
+app.get("/api/topics", getTopics);
+
+app.use("/", (req, res, next) => {
+  res.status(404).send({ msg: "Non existent endpoint" });
```

Comment:
Nicely handled. 👍 We could use `app.all` here which would cover all methods. Take a look at the bottom of the Error Handling notes for an example